google/cloud/bigquery/v2/minimal/internal/table.cc

```diff
@@ -116,165 +115,184 @@ std::string ListFormatTable::DebugString(absl::string_view name,
+      {"numActiveLogicalBytes", t.num_active_logical_bytes},
+      {"numLongTermLogicalBytes", t.num_long_term_logical_bytes},
+      {"numTotalPhysicalBytes", t.num_total_physical_bytes},
+      {"numActivePhysicalBytes", t.num_active_physical_bytes},
+      {"numLongTermPhysicalBytes", t.num_long_term_physical_bytes},
+      {"numPartitions", t.num_partitions},
+      {"numBytes", t.num_bytes},
+      {"numPhysicalBytes", t.num_physical_bytes},
+      {"numLongTermBytes", t.num_long_term_bytes},
+      {"numRows", t.num_rows},
       {"labels", t.labels},
-      {"table_reference", t.table_reference},
+      {"tableReference", t.table_reference},
       {"schema", t.schema},
-      {"default_rounding_mode", t.default_rounding_mode},
-      {"time_partitioning", t.time_partitioning},
-      {"range_partitioning", t.range_partitioning},
+      {"defaultRoundingMode", t.default_rounding_mode},
+      {"timePartitioning", t.time_partitioning},
+      {"rangePartitioning", t.range_partitioning},
       {"clustering", t.clustering},
-      {"clone_definition", t.clone_definition},
-      {"table_constraints", t.table_constraints},
+      {"cloneDefinition", t.clone_definition},
+      {"tableConstraints", t.table_constraints},
       {"view", t.view},
-      {"materialized_view", t.materialized_view},
-      {"materialized_view_status", t.materialized_view_status}};
+      {"materializedView", t.materialized_view},
+      {"materializedViewStatus", t.materialized_view_status}};
 
-  ToJson(t.last_modified_time, j, "last_modified_time");
-  ToJson(t.expiration_time, j, "expiration_time");
-  ToJson(t.creation_time, j, "creation_time");
+  ToJson(t.last_modified_time, j, "lastModifiedTime");
+  ToJson(t.expiration_time, j, "expirationTime");
+  ToJson(t.creation_time, j, "creationTime");
 }
 
 void from_json(nlohmann::json const& j, Table& t) {
   if (j.contains("kind")) j.at("kind").get_to(t.kind);
   if (j.contains("etag")) j.at("etag").get_to(t.etag);
   if (j.contains("id")) j.at("id").get_to(t.id);
   if (j.contains("type")) j.at("type").get_to(t.type);
-  if (j.contains("self_link")) j.at("self_link").get_to(t.self_link);
-  if (j.contains("friendly_name"))
-    j.at("friendly_name").get_to(t.friendly_name);
+  if (j.contains("selfLink")) j.at("selfLink").get_to(t.self_link);
+  if (j.contains("friendlyName")) j.at("friendlyName").get_to(t.friendly_name);
   if (j.contains("description")) j.at("description").get_to(t.description);
   if (j.contains("location")) j.at("location").get_to(t.location);
-  if (j.contains("default_collation"))
-    j.at("default_collation").get_to(t.default_collation);
-  if (j.contains("max_staleness"))
-    j.at("max_staleness").get_to(t.max_staleness);
+  if (j.contains("defaultCollation"))
+    j.at("defaultCollation").get_to(t.default_collation);
+  if (j.contains("requirePartitionFilter"))
+    j.at("requirePartitionFilter").get_to(t.require_partition_filter);
```

Comment:
style nit: we use `{ }` for multi-line if statements.

Feel free to not make the style changes now, and fix them by using `SafeGetTo(...)` in a follow-up PR.